### ISSUE TITLE
[a11y] Preparations check box should be selected if any preps are included

### ISSUE THREAD

**Issue by grantfitzsimmons** [specify/specify7#1401]
![image](https://user-images.githubusercontent.com/37256050/167145262-fb8a8a5e-61b7-4ce0-80ea-4cdf124b8f8a.png)

Should be checked!
